src/consensus/pbft/pbft.h

```diff
@@ -186,8 +251,23 @@ namespace pbft
       serialized::write<PbftHeader>(data_, space, hdr);
       serialize_message(data_, space, msg);
 
-      n2n_channels->send_authenticated(
-        ccf::NodeMsgType::consensus_msg, to, serialized_msg);
+      auto tmsg = std::make_unique<enclave::Tmsg<SendAuthenticatedMsg>>(
+        &send_authenticated_msg_cb,
+        std::move(serialized_msg),
+        this,
+        ccf::NodeMsgType::consensus_msg,
+        to);
+
+      uint16_t tid = 0;
+      if (enclave::ThreadMessaging::thread_count > 1)
+      {
+        tid = (to % (enclave::ThreadMessaging::thread_count - 1));
+        ++tid;
+      }
```

Comment:
This is repeated throughout the file. Could make this a function in a new thread-related file (`threading_helper.h`)?

Reply:
I put it in thread_messaging.h with the other helper functions